shell/machine-config/vmwarevsphere.vue

```diff
@@ -636,6 +637,9 @@ export default {
         }
 
         if ([_EDIT, _VIEW].includes(this.mode)) {
+          if (['tag', 'network'].includes(key) && !this.value[key]) {
+            set(this.value, key, []);
+          }
```

Comment:
I think we want to change this so that we don't mutate the value and pass an empty array to the labeled select components in the event that tag & network are null/undefined.

Reply:
that's correct, I realized that you can't have a conditional statement for `v-model`, I probably need to add a computed property for it